docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started
+
+## Prerequisites
+Before proceeding install the following requirements.
+- [Golang](https://go.dev/doc/install) (version 1.21.6)
+
+## Getting CoCos
+Get the cocos repository:
+`git clone https://github.com/ultravioletrs/cocos.git`
+
+### HAL
+Get the hardware abstraction layer from the [releases](https://github.com/ultravioletrs/cocos/releases) on the cocos repository. Two files will be required:
+- `rootfs.cpio.gz` - Initramfs
+- `bzImage` - Kernel
+
+Create two directories in `cocos/cmd/manager`, the directories are `img` and `tmp`.
+Copy the downloaded files to `cocos/cmd/manager/img`.
+
+## Starting Manager Server
+Manager is a gRPC client and needs gRPC sever to connect to. We have an example server for testing purposes in `test/manager-server`. Run the server as follows:
+
+`go run ./test/manager-server/main.go`
+
+the output should be simillar to this:
+`{"time":"2024-03-19T12:27:46.542638146+03:00","level":"INFO","msg":"manager_test_server service gRPC server listening at :7001 without TLS"}`
+
+## Running Manager
+Next we need to start manager. But first we'll need to install some prerequisites.
+
+### Vsock
+[Virtio-vsock](https://wiki.qemu.org/Features/VirtioVsock) is a host/guest communications device. It allows applications in the guest and host to communicate. In this case, it is used to communicate between manager and agent. To enable it run the following on the host:
+`sudo modprobe vhost_vsock`
+
+to confirm that it is enabled run:
+`ls -l /dev/vsock` and `ls -l /dev/vhost-vsock`
+the output should be simillar to this respectively:
+`crw-rw-rw- 1 root root 10, 121 Mar 18 14:01 /dev/vsock` and `crw-rw-rw- 1 root kvm 10, 241 Mar 18 14:01 /dev/vhost-vsock`
+
+### OVMF
+Find the ovmf code file:
+```shell
+sudo find / -name OVMF_CODE.fd
+```
+The output will be simillar to this:
+```shell
+/usr/share/edk2/x64/OVMF_CODE.fd
+/usr/share/edk2/ia32/OVMF_CODE.fd
+```
+
+Find the ovmf vars file:
+```shell
+sudo find / -name OVMF_VARS.fd
+```
+the output will be simillar to this 
+```shell
+/usr/share/edk2/x64/OVMF_VARS.fd
+/usr/share/edk2/ia32/OVMF_VARS.fd
+```
+
+### Run
+When manager client connects to server, it sends a whoAmI request after which the server sends a computation manifest. In response manager will sends logs and events from the computation both from manager and agent. To start run:
```

Comment:
`When manager client connects to server, it sends a whoAmI request after which the server sends a computation manifest.` This is confusing. What is a manager client (Is it a gRPC client to a Manager gRPC server - agent? Or the Manager is a client to some other - probably cloud - server?) and who sends requests to whom?

Reply:
manager client is manager. the server is computations service on prism or test-manager server in cocos